File: c++/RecorridoInverso.cpp
#include <iostream>
int main() {
  int numeros[] = {1, 2, 3, 4, 5};
  int tam = sizeof(numeros) / sizeof(numeros[0]);
  for (int i = tam - 1; i >= 0; i--) {
    std::cout << numeros[i] << " ";
  }
  return 0;
}

File: java/RecorridoInverso.java
public class RecorridoInverso {
    public static void main(String [] args){
        int [] numeros = {1, 2, 3, 4, 5};
        String men = "";
        for (int i = numeros.length - 1; i>= 0; i--){
            men += numeros[i] + " ";
        }
        System.out.println(men);
    }

}